seqr/views/utils/export_utils.py

```diff
@@ -52,21 +55,21 @@ def export_table(filename_prefix, header, rows, file_format, titlecase_header=Tr
         response = HttpResponse(content_type='text/tsv')
         response['Content-Disposition'] = 'attachment; filename="{}.tsv"'.format(filename_prefix)
         response.writelines(['\t'.join(header)+'\n'])
-        response.writelines(('\t'.join(map(unicode, row))+'\n' for row in rows))
+        response.writelines('\t'.join(list(map(str, row)))+'\n' for row in rows)
         return response
     elif file_format == "json":
         response = HttpResponse(content_type='application/json')
         response['Content-Disposition'] = 'attachment; filename="{}.json"'.format(filename_prefix)
         for row in rows:
-            json_keys = map(lambda s: s.replace(" ", "_").lower(), header)
-            json_values = map(unicode, row)
-            response.write(json.dumps(OrderedDict(zip(json_keys, json_values)))+'\n')
+            json_keys = [s.replace(" ", "_").lower() for s in header]
+            json_values = row
```

Comment:
theres no reason for this line of code, just change the line below it to use `rows` instead of `json_values`

Reply:
Similar to the above comment, I have to add the map back to convert non-str members to str.